product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx

```diff
@@ -86,24 +117,24 @@ Many applications execute the same SQL statement over and over again, changing o
 The following shows invoking a `PreparedStatement` that accepts an employee ID and employee name and inserts that employee information in the `emp` table:
 
 ```java
-    public void AddEmployee(Connection con) 
-    {   
-        try {    
-          Console c = System.console();   
-          String command = "INSERT INTO emp(empno,ename) VALUES(?,?)";     
-          PreparedStatement stmt = con.prepareStatement(command); 
-          stmt.setObject(1,new Integer(c.readLine("ID:")));    
-          stmt.setObject(2,c.readLine("Name:"));   
-          stmt.execute();   
-          System.out.println("The procedure successfully executed.");    
-          } catch(Exception err) {           
-            System.out.println("An error has occurred.");    
-            System.out.println("See full details below.");  
-            err.printStackTrace();  
-          } 
+    public void addEmployee(Connection con, Integer id, String name)
+    {
+        try {
```

Comment:
Again, recommend using try-with-resources, it adds no additional lines of code so doesn't impair readability.